cfgov/teachers_digital_platform/jinja2/teachers_digital_platform/activity_page.html

```diff
@@ -136,10 +136,10 @@ <h4>Activity characteristics</h4>
         <h4 class="u-mb5">National standards</h4>
         <p class="u-mb20"><small>
             (<a
-            class="a-link a-link--icon" target="_blank" rel="noreferrer noopener"
+            class="a-link" target="_blank" rel="noreferrer noopener"
             href="https://files.consumerfinance.gov/f/documents/2021_Natl_Standards_Downloadable_final.pdf"
-            ><span class="a-link__text">What are the national standards? </span>
-            <span class="a-icon">{{ svg_icon('download') }}</span></a>)
+            ><span class="a-link__text">What are the national standards?</span
+            ><span class="a-icon"> {{ svg_icon('download') }}</span></a>)
```

Comment:
```suggestion
            ><span class="a-icon">{{ svg_icon('download') }}</span></a>)
```

Reply:
Sooooooo this space is actually intentional. 

No space:
<img width="265" alt="Screenshot 2024-06-04 at 1 04 05 PM" 
src="https://github.com/cfpb/consumerfinance.gov/assets/704760/70f8e247-e810-415c-a377-4f0970f355d2">

Space:
<img width="273" alt="Screenshot 2024-06-04 at 1 04 40 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/44f33bd6-05e1-4e65-a20f-a2c48850f98a">

Reply:
Hah! I approve while disapproving!